Makefile

```diff
@@ -3,7 +3,7 @@
 hyphy:
 	echo "installing hyphy"
 	@if ! test -d ./.hyphy; then git clone http://github.com/veg/hyphy.git ./.hyphy/; fi
-	@cd ./.hyphy && git checkout master && git pull && git checkout 2.3.10 && cmake . && cd ../
+	@cd ./.hyphy && git checkout master && git pull && git checkout 2.3.10 && cmake . && make -j 4 HYPHYMP && cd ../
```

Comment:
We probably do not want to specify a hardcoded number for the `-j` argument in `make`. At most, we would have to use an environment variable or else no `-j` argument.